travis-ci/cleanup-integration-tests.sh

```diff
@@ -2,33 +2,44 @@
 
 set -e
 
+npm ci
 . ./travis-ci/set-env-vars.sh
 
-cd example || exit 1
-
-# Delete the stack if it's a nightly build
-if [ "$DEPLOYMENT" = "cumulus-nightly" ]; then
-  yarn
-  echo Delete app deployment
-
-  ./node_modules/.bin/kes cf delete \
-    --kes-folder app \
-    --region us-east-1 \
-    --deployment "$DEPLOYMENT" \
-    --yes
-
-  echo Delete iam deployment
-
-  ./node_modules/.bin/kes cf delete \
-    --kes-folder iam \
-    --region us-east-1 \
-    --deployment "$DEPLOYMENT" \
-    --yes
-
-  echo Delete app deployment
-else
-  yarn add @cumulus/common
-fi
+set +e
+(
+  set -e
+
+  cd example
+
+  # Delete the stack if it's a nightly build
+  if [ "$DEPLOYMENT" = "cumulus-nightly" ]; then
+    npm ci
+    echo Delete app deployment
+
+    ./node_modules/.bin/kes cf delete \
+      --kes-folder app \
+      --region us-east-1 \
+      --deployment "$DEPLOYMENT" \
+      --yes
+
+    echo Delete iam deployment
+
+    ./node_modules/.bin/kes cf delete \
+      --kes-folder iam \
+      --region us-east-1 \
+      --deployment "$DEPLOYMENT" \
+      --yes
+
+    echo Delete app deployment
+  else
+    rm -rf node_modules
+    npm install @cumulus/common
```

Comment:
mark and are are still not fans, but since we're passing tests, I'll allow it and give it a go to see if it can be improved.